tests/migrations/0001_initial.py

```diff
@@ -583,7 +583,7 @@ class Migration(migrations.Migration):
                 ('created', models.DateTimeField(auto_now_add=True, verbose_name='Created')),
                 ('expired', models.BooleanField(default=False)),
                 ('expiration_date', models.DateTimeField(default=concrete_datastore.concrete.models.compute_auth_token_expiry)),
-                ('last_action_date', models.DateTimeField(default=pendulum.pendulum.Pendulum.now)),
+                ('last_action_date', models.DateTimeField(default=pendulum.now)),
```

Comment:
There may be another solution to modifying migrations

Reply:
removed

Reply:
What do you mean ?